src/blog/2024-10-13/index.md

```diff
@@ -0,0 +1,117 @@
+    * `neo_cl_crosshair_circle_segments` - 30 by default, Set the segments of the circle
+
+### Ready up mechanic
+Ready up mechanic is now implemented, disabled by default. Use `neo_sv_readyup_lobby 1` server-side to enable it.
+Players can use the `!ready`/`!unready` commands to ready-up, as well as `!overridelimit` to allow for more players
+and `!playersnotready` to list players not ready.
+
+The new convars for these settings are as follows:
+
+* `neo_sv_readyup_lobby` - 0 by default, toggles the ready-up feature
+* `neo_sv_readyup_teamplayersthres` - 5 by default, the exact amount of players to ready-up and participate to start a game
+* `neo_sv_readyup_skipwarmup` - 1 by default, if ready-up feature is on and this is enabled, skip warmup state
+* `neo_sv_readyup_autointermission` - 0 by default, if disabled will not automatically enter intermission when the match ends
+
+
+### Player outlines in spectator mode
+Using the `+attack2` keybind (default `x`), can toggle between player outlines/indicators
+
+## Cloaking and visions
+
+* All visions (night vision, motion, and thermal) now have improved shaders which is now closer to OG:NT's visions
+* Cloaking material is now improved, tweaked values such as strength, speed, and distance effects more similar to OG:NT as well as when looked upon vision modes
+
+## General fixes/changes
+
+* Cleaner scope texturing and minor color adjustment
+* Workaround throwables (EX: grenades) being clipped to the ground, now can pick up grenades as normal
+* Grenade now throw closer to the center of the screen
+* Fixed general console errors, spams, and startup errors
+* Fixed wrong weapon set for the damage/killer information menu
+* Fixed reload sounds, now spatial and stops on switching/droppping
+* Prevent changing skins mid-round
+* Weapon selection available 15 seconds after round is live, 25 seconds after spawning
+* Fixed default round timings - 3m:15s for a round, 15s for pre-round freezetime
+* Round state HUD clear and reload avatar images on reload scheme
+* Allow player respawning during the first 3s of the server loaded in
+* Disable tachi duplicate muzzle flash
+* Adjust uplink HUD for larger displays (1440p/4k)
+* Fixed spawning in without a loadout
+* Stylized the chat box inspired by OG:NT
+* Lower weapons on ladder
+* Fixed center message to be more like OG:NT
+* Fixed detpack's sound
+* Fixed switching to a weapon without ammo disappearing viewmodel
+* Integrate Tony Sergi's fixes for the model animations
+* Stop camera from falling on initial team selection
+* Give knife attack some force
+* Fixed weapon stealing on very close spawns
+
+## Main menu fixes/changes
+
+* Add the ability to rebind the console key. This is mostly to help out players with keyboards that does not have the backquote key.
+* On startup, fetches the blog list from the website and opens in your default browser when clicking on one. Limited to once per day.
```

Comment:
```suggestion
* On startup, fetches the blog list from the website and opens in your default browser when clicking on one. (Only fetches once per day to prevent rate limiting).
```